polar-core/src/bindings.rs

```diff
@@ -505,11 +526,16 @@ impl BindingManager {
     }
 
     fn _variable_state(&self, variable: &Symbol) -> BindingManagerVariableState {
-        self._variable_state_at_point(variable, self.bsp())
+        self._variable_state_at_point(variable, &self.bsp())
     }
 
     /// Check the state of `variable` at `bsp`.
-    fn _variable_state_at_point(&self, variable: &Symbol, bsp: Bsp) -> BindingManagerVariableState {
+    fn _variable_state_at_point(
+        &self,
+        variable: &Symbol,
+        bsp: &Bsp,
+    ) -> BindingManagerVariableState {
+        let bsp = bsp.bsp;
```

Comment:
If all we care about is the current `BindingManager`'s `bsp`, isn't calling `self.bsp()` a bit overkill in that it will also recursively calculate & collect bsps for followers and followers followers and so on?

Reply:
Yeah, it is - but usually a binding manager will not have followers so I didn't think it's a big deal & didn't want to add some workaround to do it.